Search input box is misleading on narrow screens

When switching to narrow screens the search input shows up full width

<img width="700" alt="screen shot 2018-06-28 at 8 45 59 am" src="https://user-images.githubusercontent.com/2212006/42035070-040ef524-7ab0-11e8-9c46-f1d58d34c81c.png">

but it is not - it stays narrow, see the "clear" button

<img width="699" alt="screen shot 2018-06-28 at 8 46 10 am" src="https://user-images.githubusercontent.com/2212006/42035096-0eb9bd88-7ab0-11e8-9fbd-62f5456cd1e3.png">

thus if you try to click on the right side and type, you cannot, which is annoying

Smt like this is what I get on the Vultr instance (except the one time it worked), on my Windows computers it dies after right after INFO  Start processing

The log files are not really any more insightful then the console output...

[07:43:55] Finished 'pre:build' after 2.75 μs

> cypress-documentation@0.0.0 build /root/github/cypress-documentation
> node --stack-size=8192 ./index.js generate

disabling filter cleanup for environment development
in environment development site url is https://docs.cypress.io
NODE_ENV is: development
INFO  Start processing
Request to: http://api.jquery.com/next timed out. Ignoring this error and proceeding. Waited for 7038ms.
Request to: http://sinonjs.org/ timed out. Ignoring this error and proceeding. Waited for 9617ms.
Request to: https://api.jquery.com/nth-child-selector/ timed out. Ignoring this error and proceeding. Waited for 7654ms.
Request to: https://www.typescriptlang.org/ timed out. Ignoring this error and proceeding. Waited for 7655ms.
Request to: https://gitlab.com/cypress-io/cypress-example-docker-gitlab timed out. Ignoring this error and proceeding. Waited for 7654ms.
Request to: https://reactjs.org/ timed out. Ignoring this error and proceeding. Waited for 7655ms.
Request to: https://appveyor.com/ timed out. Ignoring this error and proceeding. Waited for 11271ms.
Request to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures timed out. Ignoring this error and proceeding. Waited for 7176ms.
Request to: https://github.com/cypress-io/cypress-test-nested-projects timed out. Ignoring this error and proceeding. Waited for 7170ms.
Request to: https://github.com/cypress-io/cypress/blob/master/packages/server/lib/config.coffee timed out. Ignoring this error and proceeding. Waited for 7169ms.
Request to: https://yarnpkg.com/ timed out. Ignoring this error and proceeding. Waited for 7170ms.
Request to: https://www.google.com/chrome/browser/desktop/index.html timed out. Ignoring this error and proceeding. Waited for 10892ms.
Request to: https://github.com/cypress-io/cypress-tutorial-build-todo-starter failed.
Request to: https://www.npmjs.com/ failed.
Request to: https://vimeo.com/242961930#t=264s failed.
Request to: https://github.com/cypress-io/cypress-watch-preprocessor timed out. Ignoring this error and proceeding. Waited for 11970ms.
Request to: https://github.com/cypress-io/cypress-documentation timed out. Ignoring this error and proceeding. Waited for 8394ms.
Request to: https://www.npmjs.com/package/npx failed.
Request to: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/stubbing-spying__window-fetch/cypress/integration/spy-stub-clock-spec.js failed.
Request to: https://github.com/adamgruber/mochawesome timed out. Ignoring this error and proceeding. Waited for 8634ms.
Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! cypress-documentation@0.0.0 build: `node --stack-size=8192 ./index.js generate`
npm ERR! Exit status 137
npm ERR!
npm ERR! Failed at the cypress-documentation@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-09-17T07_44_57_014Z-debug.log

--- a/themes/cypress/source/css/_partial/media_queries.scss
+++ b/themes/cypress/source/css/_partial/media_queries.scss
@@ -212,8 +212,6 @@
     position: relative;
     top: -6px;
     left: -120px;
-    max-width: 320px;
-    min-width: 320px;
   }
 
   #content-inner {
@@ -299,6 +297,11 @@
   .media {
     width: 100%;
     padding-right: 0;
+  }
+  
+  .algolia-autocomplete .ds-dropdown-menu {
+    max-width: 320px;
+    min-width: 320px;
   }
 }
 

--- a/themes/cypress/source/css/_partial/search.scss
+++ b/themes/cypress/source/css/_partial/search.scss
@@ -12,6 +12,9 @@
   &.on {
     display: inline-block;
   }
+  .algolia-autocomplete {
+    width: calc(100% - 30px);
+  }
 }
 
 #search-input-icon {
@@ -24,6 +27,7 @@
 }
 
 #search-input {
+  width: 100%;
   background: none;
   font-size: inherit;
   font-family: $font-sans;

